### DISCUSSION TITLE
[2기-B] 손동현

### DISCUSSION THREAD

**Discussion by nsms556** [super-corini/programmers_kdt_II#418] [2기 Monthly Project I]
##  Endpoint
  - http://gglink.iptime.org:5001/

## Github
  - https://github.com/nsms556/monthly1

##
  - 미적감각..... 누가 나한테 팔았으면 좋겠다....
  - 부트스트랩 왜 안되는데.....
  - <del>솔직히 부트스트랩 아니었으면 시간 내에 제출했는데</del>

**Comment by super-corini** [comment]
충분히 이쁩니다 😎
